packages/synapse-react-client/src/components/dataaccess/AccessRequestSubmissionTable.tsx

```diff
@@ -141,16 +141,18 @@ export const AccessRequestSubmissionTable: React.FunctionComponent<
                   </td>
                 )}
                 <td>
-                  {item.accessRequirementReviewerIds.length === 0 ? (
-                    <UserOrTeamBadge principalId={ACT_TEAM_ID} />
-                  ) : (
-                    item.accessRequirementReviewerIds.map(reviewerId => (
-                      <UserOrTeamBadge
-                        key={reviewerId}
-                        principalId={reviewerId}
-                      />
-                    ))
-                  )}
+                  <Stack gap={1}>
```

Comment:
wrap the reviewers in a flex container to create a line break between each reviewer entry:

![image](https://github.com/user-attachments/assets/4f726cf7-a248-483b-88ea-93f090313887)